utils.py

```diff
@@ -15,9 +15,12 @@ def get_item_std(item):
 
 def get_id_list(param):
     student_list = list(param[get_keyword_value('student_list')])
-    exclude_list = list(param[get_keyword_value('exclude_items')])
+    exclude_list = []
     idList = []
     responseList = []
+
+    if param.get(get_keyword_value('exclude_items')) != None:
+        exclude_list = list(param.get(get_keyword_value('exclude_items')))
```

Comment:
you can just do like that:
`exclude_list = list(param.get(get_keyword_value('exclude_items'), []))`

Reply:
if key don't exist then get returns the second argument if passed. Also, you should add a test case for this.